modules/hardware/nvidia-jetson-orin/partition-template.nix

```diff
@@ -13,11 +13,7 @@
   # TODO should this be changed when NX added
   cfg = config.ghaf.hardware.nvidia.orin;
 
-  mkSplitImages = import ./mk-split-images.nix;
-  images = mkSplitImages {
-    inherit pkgs;
-    src = config.system.build.${config.formatAttr};
-  };
+  images = config.system.build.sdImage;
```

Comment:
Some comments for the rebase, this does not need to be changed from the `config.system.build.${config.formatAttr};` (As there is now a proper nixos-generator style format module)